demo/core_demo.html

```diff
@@ -124,8 +124,8 @@
           style="display: inline-block; border: 1px solid yellow"></div>
     </div>
 
-    <script src="/demo/jquery-1.8.2.js"></script>
-    <script src="/demo/fastclick.js"></script>
+    <script src="https://code.jquery.com/jquery-1.8.3.js" crossorigin></script>
+    <script src="https://unpkg.com/fastclick@1.0/lib/fastclick.js" crossorigin></script>
```

Comment:
The reason these are included in the repo is so you can run the demo without an internet connection, which I've used many times on airplanes. So I'd rather not have this change included in this PR, as it would be a separate discussion.

Reply:
Fair enough... Then I will modify the gulp file to automatically generate generate those library files, based on the versions specified in the package file. Or we could even generate an all-in-one distribution file, including React and others... Do you need to be able to test be demo with multiple versions or having the latest version only is enough?

Reply:
Looks like my earlier reply got messed up by Github's email gateway. Latest version is fine!